envoy/api/v2/auth/BUILD

```diff
@@ -2,9 +2,18 @@ load("//bazel:api_build_system.bzl", "api_proto_library", "api_go_proto_library"
 
 licenses(["notice"])  # Apache 2
 
+package_group(
+    name = "friends",
+    packages = [
+        "-//envoy/api/v2/core",
+        "//envoy/...",
```

Comment:
Does it need to be this wide? Would prefer narrower scoping.

Reply:
Restricted it to //docs as much as possible.